testfiles/5ZoneAirCooledKeepSiteLocation.idf

```diff
@@ -0,0 +1,3503 @@
+!-Generator IDFEditor 1.34
+!-Option OriginalOrderTop UseSpecialFormat
+!-NOTE: All comments with '!-' are ignored by the IDFEditor and are generated automatically.
+!-      Use '!' comments if they need to be retained when using the IDFEditor.
+! 5ZoneAirCooled.idf
```

Comment:
Example file name. I'm not really a fan of adding new example files when this feature could be added to an existing example file. Repo size, local install size, time to test regression, etc. are impacted with each new example file. But I do understand the need for specific example files to fail, because of future code changes, for specific reasons.

Reply:
I get that we don't want to just add files without real need.  I can always remove this file and simply change an existing file.  So, would you prefer that I do that or just correct the example file name, leaving the new example file in there?

Reply:
I only have an opinion, the action is yours.

Reply:
Just committed a change that corrects the example file name shown in the comment and also adds an additional comment to explain what this file does.

Reply:
I can see both sides here.  In this case, since it is a complete change in "how the file runs" or whatever we want to call it, I don't mind it being in a standalone file.  If we were, for example, adding a new field to a chiller, we'd probably want to just modify an existing chiller test file.